.ci/Jenkinsfile

```diff
@@ -103,6 +103,7 @@ pipeline {
                     'check-packages-with-kind': generateTestCommandStage(command: 'test-check-packages-with-kind', artifacts: ['build/test-results/*.xml', 'build/kubectl-dump.txt', 'build/elastic-stack-dump/check-*/logs/*.log', 'build/elastic-stack-dump/check-*/logs/fleet-server-internal/*'], junitArtifacts: true, publishCoverage: true),
                     'check-packages-other': generateTestCommandStage(command: 'test-check-packages-other', artifacts: ['build/test-results/*.xml', 'build/elastic-stack-dump/check-*/logs/*.log', 'build/elastic-stack-dump/check-*/logs/fleet-server-internal/*'], junitArtifacts: true, publishCoverage: true),
                     'check-packages-with-custom-agent': generateTestCommandStage(command: 'test-check-packages-with-custom-agent', artifacts: ['build/test-results/*.xml', 'build/elastic-stack-dump/check-*/logs/*.log', 'build/elastic-stack-dump/check-*/logs/fleet-server-internal/*'], junitArtifacts: true, publishCoverage: true),
+                    'check-packages-custom-agent-multiservice': generateTestCommandStage(command: 'test-check-packages-custom-agent-multiservice', artifacts: ['build/test-results/*.xml', 'build/elastic-stack-dump/check-*/logs/*.log', 'build/elastic-stack-dump/check-*/logs/fleet-server-internal/*'], junitArtifacts: true, publishCoverage: true),
```

Comment:
Following this comment https://github.com/elastic/elastic-package/pull/1009#discussion_r1004497132

could this be moved to the same target as `check-packages-with-custom-agent`? 

I think this test could be considered as a different scenario when using custom-agents. It could be moved this test package to the folder `test/packages/with-custom-agent`, so it runs automatically with that test suite.

If the package is moved to that location, this task and the target in the Makefile would not be needed.

Reply:
@mrodm , just want to reconfirm what your suggestion is

1. There is no need of **new** integration test `check-packages-custom-agent-multiservice`
2. the test package which is currently under `/test/packages/custom-agent-multiservice` must go below `/test/packages/with-custom-agent`
3. Rollback the change in `.ci/Jenkinsfile` & `Makefile` 

Is my understanding correct?

Reply:
Exactly, those were the changes I was thinking of :+1: since it looks like this test could be considered a different scenario (a new package) under "with-custom-agent".
I hope there is no problem when moving the test package to the new location.